### ISSUE TITLE
Pretalx is searching for a source map that is not existing

### ISSUE THREAD

**Issue by saschafoerster** [pretalx/pretalx#1033]
When opening a browser console with f.i:
https://event.bonn.digital/bcbn2020/schedule/
I get an error 404 for: 
https://event.bonn.digital/bcbn2020/schedule/widget/pretalx-schedule.min.js.map
Same is true for 
https://pretalx.com/ncbn/schedule/
404 for: https://pretalx.com/ncbn/schedule/widget/pretalx-schedule.min.js.map
I am using Safari, macOS Big Sur, everything updated. Pretalx: newest master from git.

**Comment by rixx** [comment]
Yes, pretalx currently does not support source maps for minified JavaScript files in production mode. This is unlikely to change any time soon due to how compression and compilation works with Django and compressor.

**Comment by saschafoerster** [comment]
@rixx I am trying to embed the programm of pretalx into my website. 
I used the widget html and the script code that is provided by pretalx and embedded it in the website in the respective locations. 
When I do, they programm doesn't appear, but again I get: pretalx-schedule.min.js.map not found. 

I wonder now, why is it searching for his source maps? As I just embedded the code Pretalx provides. Can I turn this off somehow and get a working embedding? Is this on my side or is the widget-code problamtic?

Here is the actually programm-website: https://event.bonn.digital/bznb2022/schedule/widget/pretalx-schedule.min.js.map
https://bnzb.de/sessions/
